Src/Private/Get-AbrFgtSDWAN.ps1

```diff
@@ -0,0 +1,201 @@
+
+function Get-AbrFgtSDWAN {
+    <#
+    .SYNOPSIS
+        Used by As Built Report to returns SD WAN  settings.
+    .DESCRIPTION
+        Documents the configuration of Fortinet FortiGate in Word/HTML/Text formats using PScribo.
+    .NOTES
+        Version:        0.3.0
+        Author:         Alexis La Goutte
+        Twitter:        @alagoutte
+        Github:         alagoutte
+        Credits:        Iain Brighton (@iainbrighton) - PScribo module
+
+    .LINK
+        https://github.com/AsBuiltReport/AsBuiltReport.Fortinet.FortiGate
+    #>
+    [CmdletBinding()]
+    param (
+
+    )
+
+    begin {
+        Write-PScriboMessage "Discovering SD-WAN settings information from $System."
+    }
+
+    process {
+
+        Section -Style Heading2 'SD-WAN' {
+            Paragraph "The following section details SD-WAN settings configured on FortiGate."
+            BlankLine
+
+            #$sdwan = Get-fgtSystemSDWAN
```

Comment:
This should come before line 29 so that you can have an IF statement for the SD-WAN section.

`$sdwan = Get-fgtSystemSDWAN 
if ($sdwan) {
   Section -Style Heading2 'SD-WAN'
}`

Reply:
Thanks Tim,

Forget to set draft status